mqtt-client/src/main/java/com/gojek/mqtt/model/KeepAlive.kt

```diff
@@ -3,4 +3,8 @@ package com.gojek.mqtt.model
 data class KeepAlive(
     val timeSeconds: Int,
     internal val isOptimal: Boolean = false
-)
+) {
+    companion object {
+        val NO_KEEP_ALIVE = KeepAlive(timeSeconds = -1, isOptimal = false)
```

Comment:
can we use a positive value like 60 seconds for default keepalive?

Reply:
done